Packs/CortexCloud/Playbooks/Cortex_Cloud_-_AWS_S3_Bucket_Publicly_Accessible_Remediation.yml

```diff
@@ -0,0 +1,749 @@
+id: 5e89413a-0689-4a62-888a-32b184144f74
```

Comment:
The id field must match the name field, and the version field should be -1. These can be corrected automatically using the validate function in demisto-sdk.

Reply:
@shnpanw I see the `version` field has been updated.  Can you also update `id` to match the value you nave in `name` please?